web_src/less/features/heatmap.less

```diff
@@ -1,31 +1,27 @@
 #user-heatmap {
   width: 100%;
-  text-align: center;
+  font-size: 9px;
   position: relative;
   min-height: 125px;
-  align-items: center;
-  justify-content: center;
+
+  text {
+    fill: currentcolor !important;
+  }
 
   // for the "Less" and "More" legend
+  .vch__legend .vch__legend {
+    display: flex;
+    font-size: 11px;
+    align-items: center;
+    justify-content: right;
```

Comment:
How would `right` behave in other writing directions? Is this a concern for Gitea at this point?

Reply:
It's just a quick fix to get old style.

Reply:
After some testing, I think most `right` aligned solutions doesn't work well with RTL for this case.

Because the legend (the middle picture) itself is a SVG, it can not be reversed, so it's better to declare the `direction: ltr` on the `.vch__legend .vch__legend`, to make sure the `less` and `more` text never get reversed.

Reply:
I suggest to open a new issue for adressing RTL issues.